backend_new/src/dtos/application-flagged-sets/afs-meta.dto.ts

```diff
@@ -0,0 +1,35 @@
+import { ApiPropertyOptional } from '@nestjs/swagger';
+import { Expose } from 'class-transformer';
+import { IsNumber, IsOptional } from 'class-validator';
+
+export class AfsMeta {
+  @Expose()
+  @IsNumber()
+  @IsOptional()
+  @ApiPropertyOptional()
+  totalCount?: number;
+
+  @Expose()
+  @IsNumber()
+  @IsOptional()
+  @ApiPropertyOptional()
+  totalResolvedCount?: number;
+
+  @Expose()
+  @IsNumber()
+  @IsOptional()
+  @ApiPropertyOptional()
+  totalPendingCount?: number;
```

Comment:
can / should this be a computed value based on the two below? somehow seems redundant

Reply:
I think we could for sure calculate it that way, but if we need this data in more than just the partner site (like for a future reporting tool or something) its probably for the best we just calculate that on the backend and let the front end know what the value is 

hopefully that makes some sense